src/Components/NewContact/index.js

```diff
@@ -1,11 +1,104 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
+
+import { getDownloadURL, listAll, ref, uploadBytes } from "firebase/storage";
+
+import { fireStorage } from "../../Config/firebaseInit";
+
+import setContacts from "../../Services/setContacts";
+
+import styles from "./index.module.scss";
+
+const Form = () => {
+    const [state, setState] = useState({
+        firstName: "",
+        lastName: "",
+        phone: "",
+        email: "",
+        avatar: "",
+    });
+
+    const handleInputChange = (e) => {
+        setState({
+            ...state,
+            [e.target.name]: e.target.value,
+        });
+    };
+
+    useEffect(() => {
+        setContacts(state)
+        alert("Contact is ready")
+    }, [state.avatar]);
+
+    const handleSetContact = (e) => {
+        e.preventDefault();
+        const imageRef = ref(fireStorage, `contactAvatars/${e.target[4].files[0].name}`);
```

Comment:
Also as I remember I wanted to upload multiple photos, but as I can see you wrote code just for uploading one image